base/time/time_unittest.cc

```diff
@@ -515,6 +518,22 @@ TEST_F(TimeTest, LocalMidnightIsLocal) {
 }
 #endif  // BUILDFLAG(IS_FUCHSIA)
 
+#if defined(STARBOARD)
```

Comment:
No action needed here, but i think we should eventually move to `BUILDFLAG(IS_STARBOARD)` in upstream code to follow upstream OS abstraction pattern.

This can happen well after this lands in trunk though.